Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -100,7 +118,14 @@ def obfuscateApiKey(seed):
         for j in range(0, len(r), 1):
             key += seed[int(r[j]) + 2]
         return now, key
-
+    ctx = get_integration_context() or {}
+    session_id = ctx.get(SESSION_ID_KEY)
+    if session_id:
+        DEFAULT_HEADERS['cookie'] = session_id
+        try:
+            return test_module()
+        except Exception:
+            demisto.info('Zscaler encountered an authentication error with the saved session. Creating a new session.')
```

Comment:
I think I would have been better if we checked that we got the specific response that zscalar returns when the session is expired.
by minimum, log the exception error we got.

Reply:
I agree, but `http_request` raises errors as a general `Exception`. Changing this behavior will require careful handling to make sure we don't break BC.